services/horizon/internal/actions_order_book.go

```diff
@@ -82,3 +82,8 @@ func (action *OrderBookShowAction) SSE(stream sse.Stream) {
 	})
 
 }
+
+// GetTopic is a method for actions.SSE
```

Comment:
it's already known this method implements actions.SSE interface. A comment should explain what the function does along with who it belongs to: `// GetTopic returns the topic for the order book show action.` (or something similiar)